R-packages/covidcast/_pkgdown.yml

```diff
@@ -8,3 +8,43 @@ home:
 
     - text: View the COVIDcast map
       href: https://covidcast.cmu.edu/
+
+reference:
+  - title: Fetch data
+    desc: Fetch data from the COVIDcast API
```

Comment:
I might suggest: 
title: Fetch data
desc: Fetch signals from the COVIDcast API

Reply:
By the way do you think it's OK we go back and forth between "data" and "signal" (tend to use these two synonymously)?  On the one hand this is natural, on the other hand calling our signals "data" downplays the amount of work that we put into creating them.